content/general/concepts/algorithm/BigO.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: 'Big-O Notation'
+Description: 'An algorithm is a formal process used to solve a problem. To compare two algorithms, it is necessary to develop tools to evaluate algorithmic efficiency. This comparison is very important since while many algorithms might function in the same way, they may also consume a differing amount of resources. It is often best practice to limit the footprint of the algorithms that are developed and to do that, Big-O notation is to be used. '
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Algorithms'
+CatalogContent:
+  - 'paths/computer-science'
+  - 'paths/front-end-engineer-career-path'
+---
+
+Big-O notation is a form of measuring the complexity of an algorithm. This can take the form of a time complexity analysis or a space complexity analysis. At times, an algorithm can be refactored or reworked to achieve a more efficient time and space complexity. To describe the time or space complexity, two different notations are used: big omega notation and big-O. Big omega describes the best running case scenario while big-O describes the worst case scenario. 
```

Comment:
```suggestion
**Big-O notation** is a form of measuring the complexity of an algorithm. This can take the form of a time complexity analysis or a space complexity analysis. At times, an algorithm can be refactored or reworked to achieve a more efficient time and space complexity. To describe the time or space complexity, two different notations are used: big omega notation and big-O. Big omega describes the best running case scenario while big-O describes the worst case scenario. 
```